examples/gno.land/p/demo/avl/rolist/rolist.gno

```diff
@@ -0,0 +1,119 @@
+// Package rolist provides a read-only wrapper for list.List with safe value transformation.
```

Comment:
What is `list.list` exactly? It's not obvious from this package.

Reply:
it's the `gno.land/p/demo/avl/list.List` struct.